sunpy/map/mapbase.py

```diff
@@ -545,7 +583,12 @@ def carrington_longitude(self):
 
     @property
     def heliographic_latitude(self):
-        """Heliographic latitude"""
+        """
+        Heliographic latitude
+        Meta keyword : hglt_obs
```

Comment:
this is not very clear, something like `meta keyword: hglt_obs / crlt_obs / solar_b0 [in order of preference]`